### PR TITLE
COMMERCE-13217 Created Resource Requirements Card on App detail page

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
@kevenleone

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/marketplace-resource-requirements/ddm-template.ftl [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<#assign
+	filteredSpecifications=[]
+	channels=restClient.get("/headless-commerce-delivery-catalog/v1.0/channels") channelId=""
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/marketplace-resource-requirements/ddm-template.ftl:3]: Sort

```suggestion
	channelId=""
	channels=restClient.get("/headless-commerce-delivery-catalog/v1.0/channels")
	filteredSpecifications=[]
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/marketplace-resource-requirements/ddm-template.ftl [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	</#if>
+</#list>
+
+<#if (CPDefinition_cProductId.getData())??>
+	<#assign specifications = restClient.get("/headless-commerce-delivery-catalog/v1.0/channels/" + channelId + "/products/" + CPDefinition_cProductId.getData() + "/product-specifications") />
+</#if>
+
+<#if specifications?has_content && specifications.items?has_content>
+
+	<#assign cpuQuantity ="" memoryQuantity ="" />
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/marketplace-resource-requirements/ddm-template.ftl:18]:
```suggestion
	<#assign 
	    cpuQuantity = "" 
	    memoryQuantity = "" 
	/>
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
Hey @welitonpe can you simplify a bit?

**Comment by welitonpe** [comment]
> Hey @welitonpe can you simplify a bit?

Thanks Keven.

**Review by kevenleone — APPROVED** [review]
LGTM :fire

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
ci:stop

**Comment by kevenleone** [comment]
resent: https://github.com/brianchandotcom/liferay-portal/pull/146464

**Comment by kevenleone** [comment]
ci:close

**Pull request closed without merging**